_security-plugin/configuration/multi-auth.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: default
+title: Multiple option authentication
+parent: Configuration
+nav_order: 3
+---
+
+# Multiple option authentication for Dashboards sign-in
+
+You can configure the sign-in window for OpenSearch Dashboards to provide either a single option for authenticating users at sign-in or multiple options. Currently, Dashboards supports basic authentication, OpenID Connect, and SAML as the multiple options.
+
+## General steps for configuring multiple option authentication
+
+1. Decide which types of authentication to make available at sign-in.
+1. Configure each authentication type, including an authentication domain for the identification provider (IdP) and the essential settings that give each type sign-in access to OpenSearch Dashboards (see steps for each authentication type in the Configuration section of the security documentation).
```

Comment:
`(see steps for each authentication type in the Configuration section of the security documentation)`: is that possible to provide a link here for documentation reference?

Reply:
Added links for OIDC and SAML.